src/System.Runtime.Extensions/tests/Performance/Perf.StreamWriter.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Licensed to the.NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class Perf_StreamWriter
```

Comment:
I checked the tests project and we don't have any direct tests at all. If something here was broken for <=4 char, we would likely see breaks in other tests such as serialization as we verify the output is what we expect.